### PR TITLE
Dangling "pick-right-solution" references - Italian 

### LINKED ISSUE #14477: Dangling "pick-right-solution" references - Italian
This is a Bug Report
Problem:

The link from "Cluster Administration Overview" to " Picking the Right Solution" is a 404.
docs/setup/pick-right-solution/ does not exist, as it was removed as a redundant page.

Proposed Solution:
Update the stale links to point to Setup page not the "Picking the Right Solution" page
The references on this page point both too the root of the pick-right-solution page, but also to various IDs that no longer exist, there may be better pages to point those links to.

Page to Update:
https://kubernetes.io/it/docs/concepts/cluster-administration/cluster-administration-overview/

### PR DESCRIPTION
**Pull request by medmedchiheb**
this is my first PR in this project.

although there are no setup section in language/it.

Fixes #14477

### CODE REVIEW

**Comment by rlenferink** [comment]
Hi @medmedchiheb and welcome to kubernetes docs 🎉 We appreciate your contribution.

I'll assign this one to @micheleberardi as he is the main Italian translator at the moment.
/assign @micheleberardi

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
 
-Consulta le guide in [Scelta della soluzione giusta](/docs/setup/pick-right-solution/) per esempio su come pianificare, 
+Consulta le guide in [Setup](/docs/setup) per esempio su come pianificare,
```
Comment (rlenferink) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: @micheleberardi, is `Setup` correct or does it translate to something else in Italian?

Comment (micheleberardi) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: impostazioni

Comment (medmedchiheb) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: So, do i need to change 'Setup' to  'impostazioni' ? or that's fine!

Comment (rlenferink) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: @jnardiello thanks for offering your help as well (in https://github.com/kubernetes/community/pull/3745).

What do you think `Setup` should translate to (non-italian speaker here :p) ?

Comment (micheleberardi) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: I said "impostazioni" and the rest of the sentence seems fine, but it's better that @jnardiello give the final ok

Comment (jnardiello) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: I would use "Setup" in this context (it's commonly used and perfectly acceptable in italian), "Impostazioni" could be mistaken as "Settings" (that's the literal translation).

The issue in this sentence is another one: `esempio` is singular, we need to use `esempi` (plural). A correct overall sentence is:
`Consulta le guide di Setup per avere degli esempi su come pianificare, impostare e configurare cluster Kubernetes.`

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
 
-Consulta le guide in [Scelta della soluzione giusta](/docs/setup/pick-right-solution/) per esempio su come pianificare, 
+Consulta le guide in [Setup](/docs/setup) per esempio su come pianificare, 
 impostare e configurare i cluster di Kubernetes. Le soluzioni elencate in questo articolo sono chiamate *distros*.
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: `distros` -> `distribuzioni`

"Distros" is not commonly used to refer to a distribution. We call them `distribuzioni` (as per Linux distros). As we are changing this sentence I think it's worth to fix this and turn it into `*distribuzioni*`

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Presuppone una certa dimestichezza con i core Kubernetes [concetti](/docs/concep
 {{% capture body %}}
 ## Planning a cluster
```
Comment (jnardiello) [inline content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: "Planning a cluster" -> "Progettare un cluster" (1) or "Pianificare un cluster" (2)

I think option (1) is better, option (2) is still acceptable.

File: content/it/docs/concepts/cluster-administration/cluster-administration-overview.md (hunk 2 above)
Comment (medmedchiheb) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: @jnardiello  done

Comment (jnardiello) [reply content/it/docs/concepts/cluster-administration/cluster-administration-overview.md]: Thanks for the changes.

**Comment by jnardiello** [comment]
LGTM

**Comment by rlenferink** [comment]
@jnardiello many thanks for your help with reviewing the change. 
@medmedchiheb thanks for helping out to actualize our docs again 🎉 

/lgtm
/approve

**Pull request merged**